# Named exports should not use the name annotation if it is redundant to the name that would be generated by the export name (no-redundant-story-name)

Please describe the origin of the rule here.

## Rule Details

This rule aims to...

Examples of **incorrect** code for this rule:

```js
export const PrimaryButton = {
  // no need for this, as Storybook will resolve to this name already
  name: 'Primary Button',
}
```

Examples of **correct** code for this rule:

```js
export const PrimaryButton = {
  name: 'The Primary Button',
}
```

### Options

If there are any options, describe them here. Otherwise, delete this section.

## When Not To Use It

Give a short description of when it would be appropriate to turn off this rule.

## Further Reading

If there are other links that describe the issue this rule addresses, please include them here in a bulleted list.
